### PR TITLE
fix: Markdownlint cleanup for RU

### PR DESCRIPTION
**Pull request by nschonni**
Last batch of cleanup

### CODE REVIEW

**Comment by SphinxKnight** [comment]
@mdn/yari-content-ru could you please have a look?

**Comment by SphinxKnight** [comment]
If there is no veto or review in the 15 coming days, I'll merge this one after this time.

File: files/ru/web/css/_colon_nth-last-child/index.md [hunk 1]
```diff
@@ -9,15 +9,17 @@ translation_of: Web/CSS/:nth-last-child
 
 ## Описание
 
-[CSS](/ru/docs/CSS "CSS") [псевдокласс](/ru/docs/Web/CSS/Псевдо-классы "Pseudo-classes") `:nth-last-child(an+b)` находит элемент, имеющий `an+b-1` потомков после данной позиции в дереве документа, значение для `n `может быть положительным или нулевым, а также имеющий родительский элемент.
+[CSS](/ru/docs/CSS) [псевдокласс](/ru/docs/Web/CSS/Псевдо-классы) `:nth-last-child(an+b)` находит элемент, имеющий `an+b-1` потомков после данной позиции в дереве документа, значение для n может быть положительным или нулевым, а также имеющий родительский элемент.
```
Comment (sashasushko) [inline files/ru/web/css/_colon_nth-last-child/index.md:12]:
```suggestion
[CSS](/ru/docs/CSS) [псевдокласс](/ru/docs/Web/CSS/Псевдо-классы) `:nth-last-child(an+b)` находит элемент, имеющий `an+b-1` потомков после данной позиции в дереве документа, значение для `n` может быть положительным или нулевым, а также имеющий родительский элемент.
```

File: files/ru/web/css/_colon_nth-last-child/index.md [hunk 2]
```diff
@@ -9,15 +9,17 @@ translation_of: Web/CSS/:nth-last-child
...
 
 В результате, он функционирует так же, как и {{Cssxref(":nth-child")}}, кроме того, что выбирает элементы, считая в обратном порядке, с **конца** списка потомков, не с начала.
 
 Смотрите {{Cssxref(":nth-child")}} для более тщательного описания синтаксиса его аргументов.
 
 ## Синтаксис
 
-    element:nth-last-child(an + b) {стили }
+```
+element:nth-last-child(an + b) {стили }
```
Comment (sashasushko) [inline files/ru/web/css/_colon_nth-last-child/index.md:21]:
```suggestion
element:nth-last-child(an + b) { стили }
```

File: files/ru/web/css/color_value/index.md [hunk 3]
```diff
@@ -44,8 +44,9 @@ translation_of: Web/CSS/color_value
 
 - Хотя многие ключевые слова были адаптированы из [X11](https://en.wikipedia.org/wiki/X_Window_System), их значения RGB могут отличаться от соответствующего цвета в системах X11, поскольку производители иногда адаптируют цвета X11 к своему конкретному оборудованию.
 
-> **Примечание:** **Примечание:** Список принятых ключевых слов претерпел много изменений в ходе эволюции CSS:\* CSS Level 1 включал только 16 основных цветов, называемых цветами [VGA](https://en.wikipedia.org/wiki/VGA), поскольку они были взяты из набора отображаемых цветов на видеокартах VGA
+> **Примечание:** **Примечание:** Список принятых ключевых слов претерпел много изменений в ходе эволюции CSS:
```
Comment (sashasushko) [inline files/ru/web/css/color_value/index.md:47]:
```suggestion
> **Примечание:** Список принятых ключевых слов претерпел много изменений в ходе эволюции CSS:
```

**Review by sashasushko — APPROVED** [review]

**Comment by nschonni** [comment]
@sashasushko could you merge these? I can create new ones for the newly converted files once these are landed

**Pull request merged**